site-ui/src/layouts/events.hbs

```diff
@@ -0,0 +1,73 @@
+    {{#with (canonical-url)}}
+    <link rel="canonical" href="{{this}}">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,700;1,400&amp;family=Red+Hat+Display:ital,wght@0,400;0,500;0,700;1,400;1,500&amp;display=swap" rel="stylesheet">
+    {{/with}}
+{{> head-last}}
+    <script defer src="https://cdnjs.cloudflare.com/ajax/libs/list.js/1.0.2/list.min.js"></script>
+  </head>
+  <body class="{{page.layout}}">
+      <div class="container mx-auto relative">
+        {{> header }}
+        <div class="hero hero--home grad">
+            <div class="eye"></div>
+            <div id="home-content" class="text-center flex flex-center flex-column relative z2 ma-xlarge">
+                <h1>{{ page.title }}</h1>
+            </div>
+        </div>
+       <section id="events-main" class="arrow py-large">
+           <div class="inner">
+               <p class="text-center"><a href="https://events.linuxfoundation.org/cassandra-summit" target="_blank"><img src="{{{uiRootPath}}}/img/summit-2023-1296x512.png" alt="Cassandra Summit 2023" class="events-banner"/></a></p>
+               <p class="py-large text-center">
+                Cassandra events bring the community together. Everyone (whether they are an individual user, contributor, or company) is welcome to attend and help organize these events. They are an opportunity for users, enthusiasts, and community members to share their experiences working with Cassandra daily, hear talks and participate whether they are beginners or experts.
+               </p>
+               <p class="py-large text-center">
+                    Organizing an event or want to get involved? <a href="{{{ site.url }}}/_/community.html#discussions">Get in touch with us</a>!
+               </p>
+
+               <div id="all-tiles" class="grid-list-wide list pb-large">
+                    {{{page.contents}}}
+               </div>
+           </div>
+       </section>
+           
+        {{> footer}}
+      </div>
+  </body>
+<script>
+jQuery(function(){
+    var cards =  $('#all-tiles').find('.card');
+    var searchString = '';
+
+    // these will be external links, so comment:
+    // $(document)
+    // .on('click','.card',function(){
+    //     var link = $(this).find('a').attr('href');
+    //     window.open(link, '_blank');
+    //     // window.location = link ;
+    // });
+
+    // $( "#search-field" ).keyup(function() {
+    //     $('.card').removeClass('hidden');
+    //     var val = $(this).val();
+    //     val = val.toLowerCase();
+    //     searchString = val;
+    //     filterCards(searchString)
+    // });
+    var filterCards = function(searchString){
+        cards.each(function(){
+            var el = $(this);
+            var title = el.find('.discrete').text();
+            title = title.toLowerCase();
+            if(!title.includes(searchString)){
+                el.addClass('hidden');
+            }
+        });
+    }
+});
```

Comment:
What exactly is this trying to do that we can't just do in markup?

Reply:
I copied this over from a similar behaviour on the blog page.
There, a search box appears on the page to filter the entries (client-side).
At the moment, this event page has not enough entries to justify a search box, but perhaps in the future it could (or a filtering based on some geography tags of items, something like that could be useful). So I did not delete it all away but kept it for future reinstatement.

Of course this can be removed completely for cleanliness, nothing against that.

Reply:
@dchenbecker we were just trying to do the path of least resistance by replicating what the other pages already have because of the urgency of getting the page up. 🙂

Reply:
> @dchenbecker we were just trying to do the path of least resistance by replicating what the other pages already have because of the urgency of getting the page up. 🙂

Fair enough :) We might consider putting this into a common doc down the road and just include it instead of copying it, but that shouldn't block this PR